homeassistant/components/sensibo/__init__.py

```diff
@@ -1 +1,67 @@
 """The sensibo component."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+import aiohttp
+import async_timeout
+import pysensibo
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import _INITIAL_FETCH_FIELDS, DOMAIN as SENSIBO_DOMAIN, PLATFORMS, TIMEOUT
```

Comment:
Usually we just use DOMAIN instead of prefixing with the name of the integration as it's expected that working with an integration the DOMAIN will always be that of the integration